web/src/pages/mgmt/cms.tsx

```diff
@@ -63,7 +63,7 @@ const CMS = dynamic(
       // @ts-expect-error: No type definition available
       CMS.init({ CMS_CONFIG });
       // @ts-expect-error: No type definition available
-      CMS.registerWidget("write-once-read-only", WriteOnceReadOnlyControl);
+      CMS.registerWidget("write-once-read-only-no-space", WriteOnceReadOnlyNoSpaceControl);
```

Comment:
It looks like there are a couple other places where we still use `write-once-read-only`. If we're completely removing this widget, then we should update those to also use the new prop.

I just did a quick search in `config.yml` to be sure and saw a couple.

Reply:
There were two, i missed them cuz I did a find with quotes included, they all need to be swapped